fluss-common/src/main/java/com/alibaba/fluss/utils/concurrent/ConcurrentHashMapUtils.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.utils.concurrent;
+
+import org.apache.commons.lang3.JavaVersion;
+import org.apache.commons.lang3.SystemUtils;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
+
+/** Utilities that expand the usage of {@link ConcurrentHashMap}. */
+public class ConcurrentHashMapUtils {
```

Comment:
Let's move this class to `com.alibaba.fluss.utils` and rename to `MapUtils` to simplify the usage. We can adding map related utils into this class in the future.